main.py

```diff
@@ -21,7 +21,8 @@ def __init__(self):
         self.customers: List[Customer] = []
         self.active_sessions: dict[str, Customer] = {}
 
-    def create_account(self, account_type: str = "Normal", initial_balance: float = 0) -> Customer:
+    @staticmethod
```

Comment:
Why the change to a static method? Shouldn't an account be tied to a specific bank?